actors/witness.go

```diff
@@ -0,0 +1,222 @@
+package actors
```

Comment:
this class represents a witness.

It implements what a witness should do with a message. (store it, sign it, ...)